src/FakeItEasy/Configuration/IVoidAfterCallbackConfiguredConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+namespace FakeItEasy.Configuration
+{
+    /// <summary>
+    /// Provides configuration methods for methods that does not have a return value.
+    /// </summary>

```

Comment:
Oh, I forgot to mention this in my initial review: shouldn't this interface have a (slightly) different description than `IVoidConfiguration`?

Reply:
Yes. Thanks! And I'll fix `IPropertySetterAfterCallbackConfiguredConfiguration` as well.

Reply:
Fixed!